tools/istio-docker.mk

```diff
@@ -23,10 +23,14 @@ $(ISTIO_DOCKER) $(ISTIO_DOCKER_TAR):
 
 # static files/directories that are copied from source tree
 
+PROXY_BINARIES:=${ISTIO_ENVOY_DEBUG_PATH} \
```

Comment:
I don't see how the vars from bin/init.sh ever get seen here.  I do have a pending change that ports bin/init.sh into the top-level Makefile, though I'm still hung on on how/what to use to perform the download (my initial draft was to use compile an app from a short piece of go code [which was shorter and easier than the curl vs. wget detection] though Laurent suggested I try leveraging fortio instead]).

Reply:
@mattdelco The vars from Makefile are seen here, not `init.sh`. Currently we copy the vars in the Makefile and `init.sh` .... eventually we should either merge `init.sh` into the Makefile or require that it be run from the Makefile.